client/.eslintrc.json

```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "airbnb",
+    "rules": { "react/jsx-filename-extension": [0] },
+    "env": {
+        "jest": true
+    }
+}
```

Comment:
Probably a good idea to also have 

`  "parser": "babel-eslint",
`
in here. Otherwise it won't be able to lint react code.